docker/fedora/Dockerfile

```diff
@@ -3,44 +3,41 @@
 #
 #  SPDX-License-Identifier: BSD-3-Clause
 
-FROM fedora:37
+FROM fedora:39
 LABEL maintainer="Nurmukhametov, Aleksei <aleksei.nurmukhametov@intel.com>"
 SHELL ["/bin/bash", "-c"]
 
 ARG REPO=ispc/ispc
 ARG SHA=main
 
+# If you are behind a proxy, you need to configure git to use it.
```

Comment:
This comment and the comment on line 24 should be swapped.

Reply:
done